### PR TITLE
LPAL-23 - I want to see that relevant guidance on the LPA type page

### PR DESCRIPTION
**Pull request by Raffers**
## Purpose

As a first time user of the service, when I complete the "my details" page and then directed to the LPA type page, or log in for the first time after having only previously completed ‘my details’, I want to see the page from the top.

The page currently lands/anchors to the ‘What type of LPA’ question header and making it unclear to users that there is content with links above.

Fixes LPAL-23

## Approach

Creates a breadcrumb system that stores the previously accessed route, which is then passed to the RouteName twig helper function

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the LPA service_

## Checklist

* [x] I have performed a self-review of my own code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [x] I have added tests to prove my work
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-front/module/Application/src/Model/Service/Session/PersistentSessionDetails.php [hunk 1]
```diff
@@ -50,6 +46,6 @@ public function getCurrentRoute(): string {
     }
 
     public function getPreviousRoute(): string {
-        return $this->sessionDetails->previousRoute;
+        return $this->sessionDetails->previousRoute ?? 'home';
```
Comment (williamfalconeruk) [inline service-front/module/Application/src/Model/Service/Session/PersistentSessionDetails.php:49]: i guess it's reasonable to assume home is the default here?

Comment (Raffers) [reply service-front/module/Application/src/Model/Service/Session/PersistentSessionDetails.php:49]: Yer, I thought it best to set this to a default of home, with the inconsistency with how the route is populated. I did consider setting it to an empty string.

Comment (Raffers) [reply service-front/module/Application/src/Model/Service/Session/PersistentSessionDetails.php:49]: Have had to add another null check on `getCurrentRoute` as this seems to be still being populated with null somehow, despite the earlier check to make sure we have the `routeMatch`

Comment (townxelliot) [reply service-front/module/Application/src/Model/Service/Session/PersistentSessionDetails.php:49]: I think we're going to need a cypress test for this sharpish. How confident are you that we don't need it for this PR @Raffers ?

Comment (townxelliot) [reply service-front/module/Application/src/Model/Service/Session/PersistentSessionDetails.php:49]: I picked up the cypress ping test and have done a rough draft, so we can put that in as soon as we've got Nick's cypress work in.

**Comment by townxelliot** [comment]
Suggest we ensure we do a manual test of the /ping endpoint once this PR is merged to ensure it's good to go live. Then we can incorporate my automated test once Nick's work is in.

**Review by townxelliot — APPROVED** [review]
Looks good, approved.

**Review by williamfalconeruk — APPROVED** [review]

**Pull request merged**